### PR TITLE
Moving DirectorService in the separate module

### PR DESCRIPTION
**Pull request by jayeshjk**
- Needed to remove inter-app dependency between Extension app and Admin app.

### CODE REVIEW

**Comment by vinaybheri** [comment]
Validation on k8s cluster: n-2 is succeeded.

**Comment by vinaybheri** [comment]
Validation on k8s cluster: n is succeeded.

**Comment by vinaybheri** [comment]
Validation on k8s cluster: n-1 is succeeded.

**Comment by vinaybheri** [comment]
Validation on k8s cluster: n is succeeded.

File: broker/core/provisioner-services/src/DirectorService.js [hunk 1]
```diff
@@ -560,7 +560,7 @@ class DirectorService extends BaseDirectorService {
     // Lazy create of deploymentHookClient
     // Only Processes that require service lifecycle operations will need deployment_hooks properties.
     // Can be loaded on top when we modularize scheduler and report process codebase
-    const deploymentHookClient = require('../../../deployment_hooks/src/lib/utils/DeploymentHookClient');
+    const deploymentHookClient = require('../../../applications/deployment_hooks/src/lib/utils/DeploymentHookClient');
```
Comment (subhankarc) [inline broker/core/provisioner-services/src/DirectorService.js:563]: This is not good to have a dependency back to `applications`, right ?

**Pull request merged**